src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -4543,7 +4543,7 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
                         ShowWarningError(state,
                                          cCurrentModuleObject + " = " + thisVrfTU.Name + " with Fan:SystemModel is used in  " + cAlphaArgs(8) + "\"");
                         ShowContinueError(state, format("...The number of speed = {:.0R}.", double(NumSpeeds)));
-                        ShowContinueError(state, "...Multiple speed fan will be appiled to this unit. The speed number is determined by load.");
+                        ShowContinueError(state, "...Multiple speed fan will be applied to this unit. The speed number is determined by load.");
```

Comment:
Saw this typo in two new warning messages, so I fixed them. Everything else is just misc typos I fixed while I had these two cc files open.